src/main/java/nl/tudelft/ewi/devhub/server/database/AssignTAs.java

```diff
@@ -0,0 +1,54 @@
+package nl.tudelft.ewi.devhub.server.database;
```

Comment:
We could definitely add that as an enhancement but I think foremost it makes sense to be able to reassign an assigned group to someone else.